app-text/zathura-cb/zathura-cb-9999.ebuild

```diff
@@ -3,15 +3,17 @@
 
 EAPI=6
 
-inherit eutils toolchain-funcs readme.gentoo-r1 xdg
+inherit meson xdg
 
-if [[ ${PV} == *9999 ]]; then
+if [[ ${PV} == 9999 ]]
+then
 	inherit git-r3
-	EGIT_REPO_URI="https://git.pwmt.org/pwmt/zathura-cb.git"
+
+	EGIT_REPO_URI="https://git.pwmt.org/pwmt/${PN}.git"
 	EGIT_BRANCH="develop"
 else
-	KEYWORDS="~amd64 ~arm ~x86"
-	SRC_URI="https://pwmt.org/projects/zathura/plugins/download/${P}.tar.gz"
+	SRC_URI="https://pwmt.org/projects/${PN}/download/${P}.tar.gz"
+	KEYWORDS="amd64 arm x86 ~amd64-linux ~x86-linux"
```

Comment:
the KEYWORDS in the non-9999 branch of the ebuild should be ~arch by default, because
when copied to a versioned ebuild it should be unstable by default.